lib/davinci_dtr_test_kit/payer_server_groups/payer_server_adaptive_response_bundles_validation_test.rb

```diff
@@ -21,7 +21,7 @@ class PayerAdaptiveFormResponseBundlesTest < Inferno::Test
     run do
       skip_if retrieval_method == 'Static', 'Performing only static flow tests - only one flow is required.'
       test_passed = false
```

Comment:
looking at this logic, it seems like it will pass if at least one of the adaptive responses is conformant. Don't we want all of them to be conformant? I think it would look like initializing test_passed to true and then setting it in the loop to `test_passed && resource_is_valid`, right?

Reply:
I see what you mean. I think this either an oversight or initially done because the RI would return multiple questionnaires and only one was of interest anyway. But we should not test to an implementation of course. I will flip the logic